src/ZfcUser/Controller/UserController.php

```diff
@@ -61,14 +63,21 @@ class UserController extends AbstractActionController
     protected $redirectCallback;
 
     /**
-     * @param callable $redirectCallback
+     * @var ServiceLocator
      */
-    public function __construct($redirectCallback)
+    protected $serviceLocator;
+
+    /**
+     * @param callable                      $redirectCallback
+     * @param ServiceManager $serviceManager
+     */
+    public function __construct($redirectCallback, ServiceManager $serviceManager)
```

Comment:
This is BC break for sure even for me as I extended this controller and have a `parent::__construct($redirectCallback)` call in my new constructor.  You should make $serviceManager optional or remove as all others constructors